src/js/facility-locator/components/SearchControls.jsx

```diff
@@ -19,19 +27,32 @@ class SearchControls extends Component {
   handleSearch = (e) => {
     const { onSearch } = this.props;
     e.preventDefault();
+    this.toggleVisibility();
 
     onSearch();
   }
 
-  render() {
-    const style = {
-      padding: '1.5rem 1rem',
-    };
+  toggleVisibility = () => {
+    this.setState({
```

Comment:
We should also redux-ify this and use the `dispatch` actions here to update state in the store.